charts/liv-chat/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,173 @@
+{{- if $.Values.cluster.scService.networkPolicy.enabled }}
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: {{ .Values.service.name }}-egress
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    app: {{ .Values.service.name }}
+spec:
+  endpointSelector:
+    matchLabels:
+      app: {{ .Values.service.name }}
```

Comment:
@namannagar89 This will be same as [this](https://github.com/livspaceeng/helm-charts/blob/cnp_liv_chat/charts/liv-chat/templates/service.yaml#L78)

Reply:
@ankit-ls done.